templates/search.twig

```diff
@@ -377,24 +372,7 @@
 										</button>
 								</div>
 							{% endif %}
-							{% if ( pagination ) %}
```

Comment:
Since this is no longer used, we should probably update the few comments in `Search.php` that still mention it (such as [this one](https://github.com/greenpeace/planet4-master-theme/blob/master/src/Search.php#L354))?

Reply:
The comment is referring to the pagination we do use (which is what JS sends when you click the button based on what is shown).

I checked our code but nothing would something named `pagination` to the Twig context, so it's never called.

I doubt that it could even work, WP's `the_posts_pagination` function likely doesn't give the right output for the ElasticSearch query.

Reply:
Checking locally if I remove the condition, the output does have use the right total amount of pages.

But the URL doesn't work for us because our current search implementation only uses the button for pagination.

Reply:
Updated the comments to only use the general term :+1: